app/services/strategy_marketplace_service.py

```
@@ -1176,76 +1176,102 @@ async def _get_ab_test_results(self, strategy_func: str) -> Dict[str, Any]:
-                        data_quality=live_quality,
-                        created_at=strategy.created_at,
-                        last_updated=strategy.updated_at,
-                        is_active=strategy.is_active,
-                        tier="community"
+
+        async def _load_with_session(active_session: AsyncSession) -> List[StrategyMarketplaceItem]:
+            stmt = select(TradingStrategy, StrategyPublisher).join(
+                StrategyPublisher, TradingStrategy.user_id == StrategyPublisher.user_id
+            ).where(
+                and_(
+                    TradingStrategy.is_active == True,
```

Ruff (0.13.1)

Avoid equality comparisons to `True`; use `TradingStrategy.is_active:` for truth checks

Replace with `TradingStrategy.is_active`

(E712)